src/huntsman/pocs/core.py

```diff
@@ -8,15 +9,19 @@ def __init__(self, *args, **kwargs):
         self._dome_open_states = []
         super().__init__(*args, **kwargs)
 
-    def run(self, initial_next_state='starting', *args, **kwargs):
+    def run(self, initial_next_state='starting', skip_coarse_focus=False, *args, **kwargs):
```

Comment:
I'd suggest renaming it (and reverse the logic) so it's a bit more consistent from a user perspective. Either way the param should indicate it's only for the initial focus.

```suggestion
    def run(self, initial_next_state='starting', initial_focus=True, *args, **kwargs):
```